docs/book/guides/class-based-api/chapter-1.md

```diff
@@ -0,0 +1,128 @@
+---
+description: Create your first step.
+---
+
+If you want to see the code for this chapter of the guide, head over to the [GitHub](https://github.com/zenml-io/zenml/tree/main/examples/low_level_guide/chapter_1.py).
```

Comment:
is this link correct? It seems like this is the functional code with decorators at that link

Reply:
updated the link